test/integration_tests/integration_tests.py

```diff
@@ -112,3 +112,7 @@ def test_swap_inference(self):
             histo = swap_inference.test(input_, n_shots)
             histo_expected = {output: n_shots}
             self.assertEqual(histo_expected, histo)
+
+    def test_syntax_sugar(self):
+        from .tests import basis_sugar
+        self.assertEqual(bit[1](0b0), basis_sugar.sweet())
```

Comment:
Thanks for adding this integration test.

If we have a bug in the future where `{'p' >> '0', 'm' >> '1'}` acts as `id`, then this test would fail half the time and pass half the time.

I would recommend doing something like this to make this test more likely to fail when it should:

https://github.com/gt-tinker/qwerty/blob/c5f66a0c4fb5a17d27d71920f5f6a7d1baa84969/test/integration_tests/integration_tests.py#L99-L102